jobs/ingestor_syslog/spec

```diff
@@ -98,6 +99,16 @@ properties:
   logstash_ingestor.syslog.use_keepalive:
     description: Instruct the socket to use TCP keep alives
 
+  logstash_ingestor.health.disable_post_start:
+    description: Allow node to run post-start script? (true / false)
+    default: false
```

Comment:
There are some inconsistency between property name and description.
`Disable = false` should mean `enable`. However, `Allow = false` should mean `disable`.
Could you please elaborate that?